app/components/subcases/subcases-overview-header.hbs

```diff
@@ -20,12 +20,21 @@
     </Auk::Toolbar::Group>
     {{#if currentSession.isEditor}}
       <Auk::Toolbar::Group @position="right">
+        <Auk::Toolbar::Item>
```

Comment:
Deze comment slaat op de `Auk::Toolbar` component waarin deze gewrapped is:  met `@size="large"` ziet die er iets beter uit.